frameit/lib/frameit/editor.rb

```diff
@@ -239,6 +239,7 @@ def build_title_images(max_width, max_height)
         UI.verbose("Adding text '#{text}'")
 
         text.gsub! '\n', "\n"
+        text.gsub!(/(?<!\\)(')/) { |s| "\\#{s}" }
```

Comment:
This looks kind of complex, could you give some example replacements in the code base so that future us know why this is added and what it should do?